docs/source/custom-collections.rst

```diff
@@ -61,6 +64,29 @@ interface is used inside Dask.
         After computation, the results will be returned in the same layout,
         with the keys replaced with their corresponding outputs.
 
+    If more than one key is returned, all keys must be tuples sharing the same first
+    string element, which is known as the *collection name*, followed by 0+ arbitrary
+    hashables. If the output of the graph is represented by a single key, it *may* also
+    be a bare string.
+
+    These are all valid outputs:
+
+    - ``[]``
+    - ``["x"]``
+    - ``[[("y", "a", 0), ("y", "a", 1)], [("y", "b", 0), ("y", "b", 1)]``
+
+
+.. method:: __dask_layers__(self)
+
+    This method should only be implemented if the collection uses
+    :class:`~dask.highlevelgraph.HighLevelGraph` to implement its dask graph.
+
+    Returns
+    -------
+    names : tuple
+        Tuple of names of the HighLevelGraph layers which contain all keys returned by
+        :meth:`__dask_keys__`.
+
```

Comment:
Just checking, should we include the `name=` keyword in `__dask_postpersist__` here or in https://github.com/dask/dask/pull/7109?

Reply:
Here. I was convinced I added it already; fixed now.